src/codeu/chat/DerbyStore.java

```diff
@@ -110,7 +110,7 @@ public User getUser(Uuid userid) throws SQLException, IOException {
 		ResultSet user = getUser.executeQuery();
 		
 		if (user.next()) {
-			return new User(Uuid.parse(user.getString("userid")), user.getString("name"), Time.fromMs(user.getLong("creation")));
+			return new User(Uuid.parse(user.getString("userid")), user.getString("name"), user.getString("password"), Time.fromMs(user.getLong("creation")));
```

Comment:
Would it be useful if stored the names in a string array, and accessed them via that. For example, String[] userInfoColumnNames = new String[] {"userid", "name", "password", "creation"}; That way I wouldn't have to create a bunch of string variables.